src/install/local/old-quick-start.md

```diff
@@ -0,0 +1,159 @@
+#  Create an MDAI Engine instance locally
+
+>*NOTE: These docs are for MDAI Engine installation on your local machine. We are updating documentation frequently. Thank you for your patience*
```

Comment:
we should remove this file

Reply:
was keeping it for reference for the time being, and is not being indexed

Reply:
It's in version control and the git history if we need to roll back. I'll let it slide for now but I'm going to fast follow a PR and remove.